src/test/java/seedu/address/model/person/UserTest.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.TypicalUser;
+
+public class UserTest {
+    @Test
+    public void equals_sameObject_true() {
```

Comment:
Same clarification as the 2 above.

Reply:
Same object check is required by Jacoco.